internal/pkg/term/selector/selector.go

```diff
@@ -609,7 +627,9 @@ func (s *DeploySelector) deployedWorkload(workloadType string, msg, help string,
 		listDeployedWorkloads = s.deployStoreSvc.ListDeployedJobs
 		finalMessage = deployedJobFinalMsg
 	default:
```

Comment:
I would suggest make this more explicit: right now we get to default when `workloadType` is "workload". Maybe we can still error out for default and add a new case for `workloadType`?